test/test_gpu1.cpp

```diff
@@ -2737,44 +2737,64 @@ TEST_F(NVFuserTest, FusionCompoundOps_CUDA) {
   }
 }
 
-TEST_F(NVFuserTest, FusionCastOps_CUDA) {
-  Fusion fusion;
-  FusionGuard fg(&fusion);
+TEST_F(NVFuserTest, FusionFp8CastOps_CUDA) {
+  std::vector<DataType> fp8_variants(
+      {DataType::Float8_e4m3fn, DataType::Float8_e5m2});
+  std::vector<DataType> cast_targets(
+      {DataType::Double, DataType::Float, DataType::BFloat16, DataType::Half});
+
+  for (const auto& fp8_type : fp8_variants) {
+    for (const auto& src_type : cast_targets) {
+      Fusion fusion;
+      FusionGuard fg(&fusion);
 
-  TensorView* tv0 = makeSymbolicTensor(2, DataType::Half);
```

Comment:
The old test does a `half->float->half`, which would be optimized away already. So I just removed the test.